src/app/dashboard/dashboard.component.html

```diff
@@ -83,19 +83,21 @@
                 }
               </span>
             </a>
-            <a class="nav-item" mat-list-item>
-              <span
-                matTooltip="User administration"
-                matTooltipPosition="right"
-                routerLink="user-administration"
-                class="entry"
-                routerLinkActive="entry__active">
-                <mat-icon>person</mat-icon>
-                @if (!isCollapsed) {
-                  <span>User administration</span>
-                }
-              </span>
-            </a>
+            @if (getUserRole() === 'super admin') {
```

Comment:
You already declared an enum for the roles, so you should use it instead of a hard coded string